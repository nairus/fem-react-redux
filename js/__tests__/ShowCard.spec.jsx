import React from 'react';
import { shallow } from 'enzyme';
import ShowCard from '../ShowCard';

test('Landing render correctly', () => {
  const got = {
    title: 'Game of Thrones',
    year: '2011â€“',
    description: 'Nine noble families fight for control over the mythical lands of Westeros, while a forgotten race returns after being dormant for thousands of years.',
    poster: 'got.jpg',
    imdbID: 'tt0944947',
    trailer: 'giYeaKsXnsI'
  };
  const component = shallow(<ShowCard {...got} />);
  expect(component).toMatchSnapshot();
});
