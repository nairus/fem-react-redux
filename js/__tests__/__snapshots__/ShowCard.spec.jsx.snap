// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing render correctly 1`] = `
<Styled(Link)
  className="show-card"
  to="/details/tt0944947"
>
  <styled.img
    alt="Game of Thrones Show poster"
    src="/public/img/posters/got.jpg"
  />
  <div>
    <h3>
      Game of Thrones
    </h3>
    <h4>
      (
      2011â€“
      )
    </h4>
    <p>
      Nine noble families fight for control over the mythical lands of Westeros, while a forgotten race returns after being dormant for thousands of years.
    </p>
  </div>
</Styled(Link)>
`;
